recipes/pytest_loguru/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "pytest_loguru" %}
```

Comment:
The name of the python package is `pytest-loguru`, you should keep the same for the conda package.
`pip` will accept both, but not conda.

Reply:
@beenje Thanks for the review! Changes have been made.